spec/model_spec.js

```diff
@@ -2716,6 +2716,18 @@ describe('Model', function () {
           }
       }, {on: 'maxNumContext'});
 
+      this.validate('num', function() {
+        if(this.num < 0) {
+          this.addError('num', 'lowe than maximum value for num')
```

Comment:
trivial, but this error string doesn't make sense

Reply:
addressed